main.go

```diff
@@ -46,7 +46,7 @@ import (
 	operatorConfig "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/config"
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/controller/scheduledsparkapplication"
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/controller/sparkapplication"
-	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/util"
+	util "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/util"
```

Comment:
No need to alias the package name "util" to "util".

Reply:
done